src/components/Articles/Articles.jsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import { Grid } from "@mantine/core";
+import ArticleCard from "./ArticleCard.jsx";
+import Loading from "../Loading";
+
+const Articles = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articlesResponse) => {
+      setArticles(articlesResponse);
+    });
+  }, []);
+
+  if (articles.length === 0) return <Loading />;
```

Comment:
React developers largely like to stick to their conventions. We have a more conventional way of handling loading state which is contained within our notes